src/utils/payments/pawapay/pawapay.ts

```
@@ -0,0 +1,473 @@
+    } catch (error) {
+      console.error('Error checking payout status:', error);
+      throw error;
+    }
+  }
+
+  // Initiate a bulk payout
+  async initiateBulkPayout(bulkPayoutId: string, payload: any): Promise<TransactionResponse> {
+    try {
+      const path = '/bulk-payouts';
+      const headers = this.getHeaders();
+      
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)